src/libraries/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XslCompiledTransform.cs

```diff
@@ -908,7 +908,7 @@ public void LoadGeneric9(XslInputType xslInputType, ReaderType readerType, Outpu
             Assert.True(false);
         }
 
-        //[Variation(id = 11, Desc = "Load stylesheet with entity reference: Bug #68450 ")]
+        //[Variation(id = 11, Desc = "Load stylesheet with entity reference: Bug VS#68450 ")]
```

Comment:
This whole line can be deleted. I do not think anybody will be able to to find the right internal bug database that this refers to.